web/web.go

```diff
@@ -176,7 +177,7 @@ func getTemplate(name string, pathPrefix string) (*template.Template, error) {
 			return lset
 		},
 		"globalURL": func(url string) string {
-			hostname, err := os.Hostname()
+			hostname, err := getServerHostname()
```

Comment:
I'd just call it `hostname()` (using a `get` prefix is rather uncommon in Go-land in general)